doc/ref_arch/kubernetes/chapters/chapter03.md

```diff
@@ -209,71 +210,139 @@ infrastructure itself from the workloads compute resources.
 telco workloads: signalling traffic workloads and user plane traffic workloads.
 Networks used for signalling traffic are more demanding than what a standard
 overlay network can handle, but still do not need the use of user space
 networking. Due to the nature of the signalling protocols used, these type of
-networks require NAT-less communication documented in infra.net.cfg.003 and will
+networks require NAT-less communication documented in `infra.net.cfg.003` and will
 need to be served by a CNI plugin with IPVLAN or MACVLAN support. On the other
 hand, the low latency, high throughput networks used for handling the user plane
 traffic require the capability to use a user space networking technology.
 
+| Requirement | Networking Solution with Multus | Networking Solution with DANM | Networking Solution with Tungsten Fabric | Networking Solution with NSM |
+|---|---|---|---|---|
+| The overlay network encapsulation protocol needs to enable ECMP in the underlay (`infra.net.cfg.002`) | Supported via the additional CNI plugin | Supported via the additional CNI plugin | Supported | TBC |
+| NAT (`infra.net.cfg.003`) | Supported via the additional CNI plugin | Supported | Supported | TBC |
+| Network Policies (Security Groups) (`infra.net.cfg.004`) | Not supported | Not supported <sub>(1)<sub> | Supported | Supported via a CNI Network Plugin that supports Network Policies |
+| SFC support (`infra.net.cfg.005`) | Not relevant | Not relevant | Not relevant | Not relevant |
+| Traffic patterns symmetry (`infra.net.cfg.006`) | Not relevant | Not relevant | Not relevant | Not relevant |
+| Network resiliency (`req.inf.ntw.01`) | Supported | Supported | Supported | Supported |
+| Centrally administrated and configured (`req.inf.ntw.03`) | Supported via Kubernetes API Server | Supported via Kubernetes API Server | Supported via Kubernetes API Server | Supported via Kubernetes API Server |
+| Dual stack IPv4 and IPv6 for Kubernetes workloads (`req.inf.ntw.04`) | Supported via the additional CNI plugin | Supported | Supported | Supported |
+| Integrating SDN controllers (`req.inf.ntw.05`) | Supported via the additional CNI plugin | Supported via the additional CNI plugin | TF is an SDN controller | TBC |
+| More than one networking solution (`req.inf.ntw.06`) | Supported | Supported | Supported | Supported |
+| Choose whether or not to deploy more than one networking solution (`req.inf.ntw.07`) | Supported | Supported | Supported | Supported |
+| Kubernetes network model (`req.inf.ntw.08`) | Supported via the additional CNI plugin | Supported via the additional CNI plugin | Supported | Supported via the default CNI plugin |
+| Do not interfere with or cause interference to any interface or network it does not own (`req.inf.ntw.09`) | Supported | Supported | Supported | Supported |
+| Cluster wide coordination of IP address assignment (`req.inf.ntw.10`) | Supported via the additional CNI plugin | Supported | Supported | Supported via IPAM CNI plugin |
```

Comment:
It should be same for Multus and Networking Solution with NSM

Reply:
Updated in https://github.com/cntt-n/CNTT/commit/62eef1a8aaf0743f1b9895ffb8472a34237f4f27